### PR TITLE
Remove deprecated stuff

### PR DESCRIPTION
**Pull request by carusogabriel**
Following #1420

### CODE REVIEW

**Comment by carusogabriel** [comment]
Gonna search and fix it!

**Comment by carusogabriel** [comment]
Some if failing, but I couldn't figure out what :disappointed:

**Comment by p365labs** [comment]
yes 'cause there are tests on the this->_log method change it to this->_logger, don't remove it

**Comment by ruflin** [comment]
@carusogabriel Could you rebase on top of master? I just merged https://github.com/ruflin/Elastica/pull/1431

File: lib/Elastica/Client.php [hunk 1]
```diff
@@ -689,8 +689,6 @@ public function request($path, $method = Request::GET, $data = [], array $query
         } catch (ConnectionException $e) {
             $this->_connectionPool->onFail($connection, $e, $this);
 
-            $this->_log($e);
```
Comment (stof) [inline lib/Elastica/Client.php]: shouldn't this be replace by a `$this->_logger->error()`  call ? Overriding the method was deprecated. But now, you removed the logging entirely.

Comment (carusogabriel) [reply lib/Elastica/Client.php]: @ruflin What you say? I'll update this PR today, I forgot it :sweat_smile:

Comment (ruflin) [reply lib/Elastica/Client.php]: +1 on the change suggested by @stof

**Comment by ruflin** [comment]
@carusogabriel Just saw now that you pushed a new commit. Could you also add a changelog? And it seems to break travis somehow.

**Comment by ruflin** [comment]
@carusogabriel Can you have a look at the comment here https://github.com/ruflin/Elastica/issues/1374#issuecomment-366554090 from @greg0ire to check into which "category" your changes fall?

**Comment by ruflin** [comment]
@carusogabriel Can you share some background on the closing of the PR?

**Comment by carusogabriel** [comment]
@ruflin Sure. I closed as I wasn't able to solve all issues and have green tests. I believe you know the library better than me to finish it :smile:

**Comment by ruflin** [comment]
@carusogabriel I see. I was happy that you took that one on :-) If you find time, perhaps you can split up the PR into the parts that you got working so we can get at least these in?

**Pull request closed without merging**